### PR TITLE
Devops/switch ports (completes PRA-59)

### PR DESCRIPTION
**Pull request by rebekahcallkacz**
## One Line Description
Switch API port from 8000 to 3000. Switch React app port from 3000 to 3005. Add better API error handling

## Requirements
* Front end app should run on port 3005
* When the API is not working, no TypeScript errors should show up
* Response from API should be black if successful and red if not.

## Notes
We'll want more sophisticated error handling later, but this should make things a little easier for now.

## Test Steps
* Add `.env.local` file to top level of repository locally
* Add variable `PORT = 3005` to the env file
* Run back end
* Run front end
* Verify that app loads successfully and renders message about response from API route in black
* Shut down back end
* Refresh front end app in browser - verify that the message about the API response is now in red and says "network error"

## Checklist

- The Linear ID is in the PR title
- My code follows best practices
- Documentation is included where needed

### CODE REVIEW

File: src/util/index.js [hunk 1]
```diff
@@ -1,25 +1,26 @@
...
+    return data;
+  } catch (error) {
+    console.log(error, `error - getAllData in ${url} route`);
+    throw error;
+  }
+};
 
-  export {getData, getAllData};
\ No newline at end of file
+export { getData, getAllData };
```
Comment (rebekahcallkacz) [inline src/util/index.js:26]: No actual changes to this file - just standardizing spacing

File: src/App.tsx [hunk 2]
```diff
@@ -1,19 +1,22 @@
...
+  const [textColor, setTextColor] = useState<string>("black");
 
   useEffect(() => {
-    (async () => {
-      const myData = await getAllData(URL);
-      setMessage(myData.data);
-    })();
+    getAllData(URL)
+      .then((response) => setMessage(response.data))
+      .catch((error) => {
```
Comment (rebekahcallkacz) [inline src/App.tsx:16]: Catching the error here will ensure that the typescript error you've been seeing doesn't show up when the API isn't working. Instead, the error message will display on the page.

**Review by AnnaSolovykh — APPROVED** [review]
Looks great! Thank you!

**Review by AnnaPestova1 — APPROVED** [review]
Thank you

**Pull request merged**